app-modules/ai/src/Providers/AiServiceProvider.php

```diff
@@ -87,5 +90,7 @@ protected function registerObservers(): void
         AiAssistant::observe(AiAssistantObserver::class);
         AiMessage::observe(AiMessageObserver::class);
         Prompt::observe(PromptObserver::class);
+        PromptUse::observe(PromptUsesObserver::class);
+        AiThread::observe(AithreadObserver::class);
```

Comment:
If we do remove the Observers like I mentioned above, let's ensure we remove this.

Reply:
Done